flink-yarn/src/main/java/org/apache/flink/yarn/Utils.java

```diff
@@ -187,6 +226,27 @@ public static void deleteApplicationFiles(final Map<String, String> env) {
 			LOG.debug("No yarn application files directory set. Therefore, cannot clean up the data.");
 		}
 	}
+	/**
+	 * Creates a YARN resource for the remote object at the given location.
+	 *
+	 * @param remoteRsrcPath	remote location of the resource
+	 * @param resourceSize		size of the resource
+	 * @param resourceModificationTime last modification time of the resource
+	 *
+	 * @return YARN resource
+	 */
+	private static LocalResource registerLocalResource(
+		Path remoteRsrcPath,
+		long resourceSize,
+		long resourceModificationTime) {
+		LocalResource localResource = Records.newRecord(LocalResource.class);
+		localResource.setResource(ConverterUtils.getYarnUrlFromURI(remoteRsrcPath.toUri()));
+		localResource.setSize(resourceSize);
+		localResource.setTimestamp(resourceModificationTime);
+		localResource.setType(LocalResourceType.FILE);
+		localResource.setVisibility(LocalResourceVisibility.APPLICATION);
+		return localResource;
+	}
```

Comment:
This method is the same as in the reverted commit - instead of adding it again, do you think, it makes sense to just not revert this part?
Otherwise please fix the indentation of the parameter list.

Reply:
The revert is suggested by Till in the Jira. Could you be specific on indentation on which line? It seems following the convention.

Reply:
your revert commit is actually already a partial revert - these lines are basically removed by the revert and then added back - unnecessarily. History (and authorship) would be clearer without this.